### ISSUE TITLE
Accessibility (WCAG level AA) "Required" image on form pages lacks sufficient contrast and isn't adaptive to OS settings.

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1173]
1.  Download Paciello Group's Contrast Analyser from http://www.paciellogroup.com/resources/contrastAnalyser.  This tool measures foreground-to-background contrast ratios using the same algorithm defined by the W3C's Web Content Accessibility Guidelines.
2.  Navigate to https://nuget.org/users/account/LogOn.  (This issue occurs on all forms on the site.)

ISSUE 1:

3.  Compare the contrast ratio of the text in the "Required" image against its background.

Result:  It's 3.33 : 1.  
Expected:  This is below the threshold of 4.5:1 given in WCAG.

ISSUE 2:

On a Windows machine, toggle high-contrast mode by pressing Left Alt + Left Shift + Print Screen.

Result:  Unlike the rest of the site UI, the image isn't adaptive, and will still have the same appearance, even though the rest of the page has changed to the high-color palette.
Expected:  Especially for a glyph like this that's meant to provide instructions, we want it to be adaptive to the user's settings.  We should replace the image with actual markup that illustrates the same point.

**Comment by johnataylor** [comment]
We have a new gallery design.
